### PR TITLE
Include jsonnet-language-server

### PR DESCRIPTION
**Pull request by zalegrala**
Here we include an initial configuration for the jsonnet-language-server.

https://github.com/jdbaldry/jsonnet-language-server

### CODE REVIEW

**Comment by zalegrala** [comment]
https://github.com/williamboman/nvim-lsp-installer/pull/240

File: lua/lspconfig/jsonnet.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.jsonnet = {
+  default_config = {
+    cmd = { 'jsonnet-language-server' },
+    filetypes = { 'jsonnet', 'libsonnet' },
+    root_dir = function(fname)
+      return util.root_pattern 'go.work'(fname) or util.root_pattern('go.mod', '.git')(fname)
```
Comment (zalegrala) [inline lua/lspconfig/jsonnet.lua:9]: This may not be correct.

Comment (zalegrala) [reply lua/lspconfig/jsonnet.lua:9]: I want this to launch from the directory that vim was launched from.  Does someone have a pointer for this?

Comment (mjlbach) [reply lua/lspconfig/jsonnet.lua]: We're not going to support that in lspconfig until I have time to implement scratch server support.

File: lua/lspconfig/jsonnet.lua [hunk 2]
```diff
@@ -0,0 +1,22 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.jsonnet = {
+  default_config = {
+    cmd = { 'jsonnet-language-server' },
+    filetypes = { 'jsonnet', 'libsonnet' },
+    root_dir = function(fname)
+      return util.root_pattern('jsonnetfile.json')(fname) or util.find_git_ancestor(fname) or util.path.dirname(fname)
```
Comment (lithammer) [inline lua/lspconfig/jsonnet.lua:9]:
```suggestion
      return util.root_pattern('jsonnetfile.json')(fname) or util.find_git_ancestor(fname)
```

**Comment by mjlbach** [comment]
wondering if we should name it jsonnet_ls, I've started recommending people do this when the name of the server is the language name

**Comment by zalegrala** [comment]
@mjlbach Do you mean rename the `jsonnet.lua` file?

**Comment by williamboman** [comment]
> @mjlbach Do you mean rename the `jsonnet.lua` file?

Renaming the file and changing the table key ought to do it!

```diff
- configs.jsonnet = {
+ configs.jsonnet_ls = {
```

**Comment by zalegrala** [comment]
Got it, I've renamed the file and the config reference.

**Pull request merged**